python-suse-cloud-image-name-parser.spec

```diff
@@ -45,13 +47,13 @@ BuildArch:      noarch
 
 %install
 %pyproject_install
-%{python_expand %fdupes %{buildroot}%{$python_sitelib}}
+%python_expand %fdupes %{buildroot}%{_sitelibdir}
```

Comment:
`%python_expand`  is not needed, there is nothing to expand everything is already determined, `%fdupes %{buildroot}%{_sitelibdir}` is sufficient.